docs/hacking-guide/en/tests.md

```diff
@@ -22,20 +22,20 @@ The broker is comprised of POJOs so it's simple to configure and run a broker in
 Even complex test-cases involving multiple clustered brokers are relatively easy to write. Almost every test in the 
 test-suite follows this pattern - configure broker, start broker, test functionality, stop broker.
 
-The test-suite uses JUnit to manage test execution and life-cycle.  Most tests extend [`org.apache.activemq.artemis.tests.util.ActiveMQTestBase`](../../../artemis-server/src/test/java/org/apache/activemq/artemis/tests/util/ActiveMQTestBase.java)
+The test-suite uses JUnit to manage test execution and life-cycle.  Most tests extend [`org.apache.activemq.artemis.tests.util.ActiveMQTestBase`](https://github.com/apache/activemq-artemis/blob/master/artemis-server/src/test/java/org/apache/activemq/artemis/tests/util/ActiveMQTestBase.java)
```

Comment:
@clebertsuconic thanks, understood. 

My initial thought was to point them to the API documentation already present on the website, but some of the code mentioned on the documentation does not appear there. 

Do you have any suggestion about where to point these links or how to handle it? I can adjust these links to point to the correct location or I can try to generate the docs for the missing classes ...